### ISSUE TITLE
Fix codacy code quality issues

### ISSUE THREAD

**Issue by faruktoptas** [faruktoptas/FancyShowCaseView#171]
Fix issues listed on codacy.
https://app.codacy.com/manual/faruktoptas/FancyShowCaseView/issues

**Comment by eric-muchiri** [comment]
I can try issues on the readme.md

**Comment by lhllyra** [comment]
Hey, so I have been trying to get the project up and running on Intellij to take a closer look at the code and see if I could be of any help. 

I wasn't able to get too far though, as I have run into some problems when trying to build the project through gradle.

I tried to build the program ("gradle build" command) to run the debugger and got this build failed message: 

Failed to install the following Android SDK packages as some licences have not been accepted.
     patcher;v4 SDK Patch Applier v4
     build-tools;28.0.3 Android SDK Build-Tools 28.0.3
     emulator Android Emulator
     tools Android SDK Tools
     platforms;android-28 Android SDK Platform 28
     platform-tools Android SDK Platform-Tools


I have tried all kinds of fixes I was able to find around the web, but for the life of me, I can't seem to build the project.

I found some interesting fixes around [here](https://github.com/jitpack/jitpack.io/issues/3687), but it still didn't cut it. Has this happened to anyone around here too?
